lib/redis_memo/memoizable/invalidation.rb

```diff
@@ -28,8 +45,6 @@ module RedisMemo::Memoizable::Invalidation
   @@invalidation_queue = Queue.new
 
   def self.bump_version_later(key, version, previous_version: nil)
-    RedisMemo::DefaultOptions.logger&.info("[received] Bump memo key #{key}")
-
```

Comment:
what's the motivation for removing these log lines? I feel like they're useful when developing things locally; could we also turn these on in prod?

Reply:
We will still have the same information during local dev; it's just moved from STDOUT to the tracing service (datadog) UI.

Logging with `[received]` `[performed]` might be a little too verbose anyway